docs/EXTRAS_ORM.rst

```diff
@@ -0,0 +1,137 @@
+Miscellaneous 
+=============
+
+The items listed below are optional and intended to enhance 
+integration between Zend Framework and Doctrine 2.
+
+ObjectExists Validator and  NoObjectExists Validator
+----------------------------------------------------
+
+ObjectExists and NoObjectExists are validators similar to
+`Zend Validators <https://framework.zend.com/manual/2.4/en/modules/zend.validator.html>`_. 
+You can pass a variety of options to determine validity.
+The most basic use case requires an entity manager, an entity, and
+a field. You also have the option of specifying a query\_builder Closure
+to use if you want to fine tune the results.
+
+.. code:: php
+
+    <?php
+    $validator = new \DoctrineModule\Validator\NoObjectExists([
+        // object repository to lookup
+        'object_repository' => $serviceLocator->get('doctrine.entitymanager.orm_default')   
```

Comment:
In the previous md-file, a FQCN was used for the service id. I assume you changed this to show support multiple entity managers.

Reply:
In ZF2 and Doctrine you should never get the object manager from the FQCN because ZF2 + Doctrine uses aliases for object manager locator names.  

Correcting this is an improvement of documentation which is outdated.